Blackfynn/data-portal
# Download dataset dialog #188u4z

# Description
The purpose of this PR is to add a download dataset dialog to dataset page.

![127 0 0 1_5000_browse_(Laptop with HiDPI screen) (2)](https://user-images.githubusercontent.com/1493195/61401980-21fd6880-a8a0-11e9-9d49-d5574514a3c1.png)



## Type of change


# How Has This Been Tested?

- Open a [dataset](http://127.0.0.1:5000/browse/#/datasets/12) from the browse menu
- Click on the "Get dataset" button 
- Dialog should be visible
- Verify data in the dialog is correct
- Click on "Download Dataset"
- Dataset should be downloading


# Checklist:
